spec/values/calculation/round/three_arguments.hrx

```diff
@@ -0,0 +1,247 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> positive/input.scss
```

Comment:
It's confusing that the name of this test implies that it's not specific to a given strategy, when in fact all three-argument `round()` calls _have to be_ specific to a given strategy. I think it would make more sense to have all fully-simplified tests be in strategy-specific directories, and test each relevant combination for each strategy.

Reply:
I like the current structure you have, where you have strategy subdirectories that fully exercise each strategy but have tests that aren't particularly strategy-specific outside those subdirectories.